src/token/token.controller.ts

```diff
@@ -96,4 +142,23 @@ export default class TokenController {
     );
     res.json(result.data).send();
   }
+
+  @Post('/schema/upload')
+  @UseInterceptors(FileInterceptor('upload'))
+  async uploadSchema(@UploadedFile() file: Express.Multer.File, @Res() res) {
+    await fs.writeFile(join(process.cwd(), 'schema/token', file.originalname), file.buffer);
+    res.status(302).redirect('/api/token/decode');
+  }
+
+  @Get('/schema/download')
+  downloadSchema(@Query('schema') schema_s: string, @Res() res: Response) {
+    const file = createReadStream(join(process.cwd(), 'schema/token', schema_s));
+    file.pipe(res);
+  }
+
+  @Get('/schema/delete')
+  async deleteSchema(@Query('schema') schema_s: string, @Res() res: Response) {
+    await fs.unlink(join(process.cwd(), 'schema/token', schema_s));
+    res.status(302).redirect('/api/token/decode');
+  }
```

Comment:
As an alternative to an upload, we could just let the user input a path to his or her schema, so the application can access it during execution. This would remove the need for the upload and the delete actions. 
What do you think about this idea?

Reply:
I think it's a good idea. I added the option to choose a local file.
I still kept the options to upload/download schemas. I think it is quite handy for a quick selection.